### DISCUSSION TITLE
Concept Art

### DISCUSSION THREAD

**Discussion by gamerevar** [gamerevar/proyecto-comunitario-1-gd#62] [Archivados]
Por favor, cuando suban el concept art, pongan el $id de la _card_ de la cuál sacaron la tarea:

![](https://i.ibb.co/CW4jNC4/firefox-g-XZ19-DVGMB.png)

Pueden hacer **drag & drop** del archivo y se sube automaticamente!

## Subir el concept aca: [Concept Art Refence Sheet](https://github.com/gamerevar/proyecto-comunitario-1-gd/discussions/63)

**Comment by ignacio-otu** [comment]
![soldados](https://user-images.githubusercontent.com/75650122/103228507-c26a1000-490f-11eb-90fa-db6f239d6d1e.jpg)

**Reply by ignacio-otu** [reply]
https://open.codecks.io/proyecto-comunitario-1-gd/decks/6-arte/card/12c-soldados

**Reply by gamerevar** [reply]
> https://open.codecks.io/proyecto-comunitario-1-gd/decks/6-arte/card/12c-soldados

Tiralos en el tema que creo @Ecoabismo!

**Comment by ignacio-otu** [comment]
![soldados2](https://user-images.githubusercontent.com/75650122/103299895-5c978a00-49dc-11eb-9cab-312ca98114e7.jpg)

tomando en cuenta la anatomía de del diseño pixel de Ferchiman

**Reply by gamerevar** [reply]
Agregados!

> ![soldados2](https://user-images.githubusercontent.com/75650122/103299895-5c978a00-49dc-11eb-9cab-312ca98114e7.jpg)
> 
> tomando en cuenta la anatomía de del diseño pixel de Ferchiman

**Comment by Ecoabismo** [comment]
Bueno hoy avanzamos banda: Acá está la base de lo que armamos en la llamada:
Si arman los concepts como el golemcito en psd o incluso a mano, lo ideal es que tenga ya la perspectiva de idle. Por ejemplo uno puede hacer en pixel art un concept que haya armado otro etc.
La seguimos luego!

![BaseForCharacter](https://user-images.githubusercontent.com/40045009/103327001-8d55de80-4a31-11eb-866e-7b67267ac162.png)
![BaseGolem](https://user-images.githubusercontent.com/40045009/103327003-8dee7500-4a31-11eb-86ef-769446632edd.png)
![BaseWithCharacter](https://user-images.githubusercontent.com/40045009/103327004-8e870b80-4a31-11eb-96ef-93f2bc9a2c1f.png)
![Golem](https://user-images.githubusercontent.com/40045009/103327005-8e870b80-4a31-11eb-8fe8-f54461c200a2.png)
![imagen_2020-12-29_202640](https://user-images.githubusercontent.com/40045009/103327007-8f1fa200-4a31-11eb-8a43-ccf040cdc4d0.png)

**Comment by Ecoabismo** [comment]
Van las bases del Golem y Soldado.

![CharacterMagicSwatRifle](https://user-images.githubusercontent.com/40045009/103413246-832e0000-4b57-11eb-9661-8c76d788c1c5.png)
![BaseGolem](https://user-images.githubusercontent.com/40045009/103413248-845f2d00-4b57-11eb-9549-a542bf46e10b.png)
![CharacterMagicSwat](https://user-images.githubusercontent.com/40045009/103413250-845f2d00-4b57-11eb-8d91-84a64bf4b93c.png)

**Reply by Ecoabismo** [reply]
![CharacterMagicSwatRifle](https://user-images.githubusercontent.com/40045009/103413314-bff9f700-4b57-11eb-8c45-6e19c9a6c27a.png)
Aca estaría con la luz de la teconogía cristal.
